service/sync.go

```diff
@@ -48,6 +48,20 @@ func (s *SyncImpl) Resolve(forceUpdate bool) error {
 	newdeps := make([]dependency.ProtoDepDependency, 0, len(protodep.Dependencies))
 	protodepDir := filepath.Join(s.userHomeDir, ".protodep")
 
+	if cleanupCache {
+		files, _ := ioutil.ReadDir(protodepDir)
```

Comment:
Please add error checking. 
And also, don't nest too deep.

For example, as follows:

```go
		files, err := ioutil.ReadDir(protodepDir)
		if err != nil {
			return err
		}
		for _, file := range files {
			if file.IsDir() {
				dirpath := filepath.Join(protodepDir, file.Name())
				if err := os.RemoveAll(dirpath); err != nil {
					return err
				}
			}
		}
```

Reply:
Thanks!
update f26135e

Because we don't know if the `.protodep` directory exists, Add a Stat() call before ReadDir().
It not known exist `.protodep` directory, add Stat() call before ReadDir().
Please review. 🙏

Reply:
I see. This is okay.